Tribler/Test/Core/CreditMining/test_creditmining_sys.py

```diff
@@ -164,11 +163,19 @@ def setUpFileServer(self, port, path):
         self._logger.debug("Listen to port %s, factory %s", port, factory)
         self.file_server = reactor.listenTCP(port, factory)
 
-    def check_feed_err(self, dummy_fail, dummy_rss):
+    def check_feed_err(self, src, err_defer=None):
         """
         check whether a feed is producing error or not
         """
-        self.error_event.set()
+        if err_defer is None:
+            err_defer = defer.Deferred()
+
+        if self.is_error_fired:
+            err_defer.callback(True)
+        else:
+            reactor.callLater(1, self.check_feed_err, src, err_defer)
```

Comment:
Because I want to 'wait' until `_on_error_rss` is fired by `BoostingManager` errback. As the load function in BoostingManager not returning a deferred object, I can't find other way to check this.